salt/_runners/metalk8s_saltutil.py

```diff
@@ -61,11 +62,50 @@ def condition_reached(minions, attempts):
             'error': error_message
         }
 
+    # Waiting for startup state to finish
```

Comment:
`saltutil.running` shows more than just "startup states", IIUC, it's actually showing all the running jobs on the minion (which are module executions, not states, so we would get a bunch of `{"fun": "state.sls", "jid": ..., "pid": ..., "tgt": ...}` in `state_running`).

I think we could adjust comments and log messages to mark them as "running jobs" instead of "startup state".
It would also make sense to show info about the running jobs themselves:

```python
log.debug(
    "[Attempt %d/%d] Waiting for running jobs to complete: %s",
    attempts,
    retry,
    ' - '.join(
        'Jobs on minion "{minion}": {jobs}'.format(
            minion=minion,
            jobs=', '.join(
                '"{job[fun]}" ({job[pid]})'.format(job=job)
                for job in minion_jobs
            )
        )
        for minion, minion_jobs in running_jobs
    ),
)
```

_Note that I used `log.debug`, not sure this kind of "waiting" log warrants to be printed by default..._